.github/workflows/config/spelling_allowlist.txt

```diff
@@ -98,6 +101,7 @@ executables
 inlining
 instantiation
 instantiations
+json
```

Comment:
JSON should be spelled in all uppercase throughout the codebase so this should be removed.

Reply:
Removed and updated. I _think_ there was only one place with `token.json` in comments, let's see what CI run says.